hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/commit/AbstractITCommitProtocol.java

```diff
@@ -403,6 +403,30 @@ public JobData(Job job,
       this.committer = committer;
       conf = job.getConfiguration();
     }
+
+    public Job getJob() {
+      return job;
+    }
+
+    public JobContext getJContext() {
```

Comment:
use JobContext and TaskContext in getters, even if field names aren't as informative

Reply:
actually i'm not worrying about this; will merge as is.